### ISSUE TITLE
A11Y : Content Rows

### ISSUE THREAD

**Issue by jcsparks** [CuBoulder/tiamat-theme#907]
Examples here : https://live-ucbdev-sandbox-a11y.pantheonsite.io/content-rows

Most options – teasers, teasers alternate, and features – had the same issue with reading order. The image appears prior to the heading. While these images are more decorative, this could be a significant issue depending on what image is provided and how it associates with the heading level 3. (Significant)

 - Make the order consistent : Title, image, text and use flexbox and pseudo classes to adjust visually as needed.

**Comment by jcsparks** [comment]
Note that if the images can become links we need to ensure that duplicate links have a `role="presentation"` and `aria-hidden="true"` set on the anchor tag -- the title should be the main link for the content.
